tests/exchange_translator_tests.cc

```diff
@@ -97,6 +98,25 @@ TEST(ExXlateTests, NegPref) {
   EXPECT_EQ(graph->arcs().size(), 0);
 }
 
+TEST(ExXlateTests, ZeroPref) {
```

Comment:
Is this test checking that zero pref arcs get removed? If so, could you throw in a doc comment that says this - or that says whatever the test is checking.